meta-aos-rpi-domd/recipes-core/images/aos-image-rpi.bb

```diff
@@ -3,15 +3,27 @@ SUMMARY = "Aos image for Raspberry Pi devices"
 require recipes-core/images/rpi5-image-xt-domd.bb
 require recipes-core/images/aos-image.inc
 
+inherit extrausers
+
 IMAGE_INSTALL:append = " \
     packagegroup-core-ssh-openssh \
     tzdata \
+    sudo \
 "
 
 IMAGE_INSTALL:append = " \
     aos-messageproxy \
 "
 
+EXTRA_USERS_PARAMS = "\
+    useradd -u 1200 -d /home/aos -s /bin/sh -p '*' aos; \
```

Comment:
Why do you need to specify all these parameters explicitly? What about 'useradd aos;'?